portfolio/src/main/java/com/google/sps/data/LoginInfo.java

```diff
@@ -0,0 +1,13 @@
+package com.google.sps.data;
+
+/** Class containing user login info */
+public final class LoginInfo {
+  /** Log-in link if user is logged out, log-out link if the user is logged in.*/
+  public final Boolean loggedIn;
+  public final String link;
```

Comment:
Private instance variables might make more sense here, so that someone can't start toggling users' login statuses and where they get redirected to if you don't want to

Reply:
done! thanks